### ISSUE TITLE
Case caption on new case page needs to be an H1 tag

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#4788]
Using a div for this is bad for accessibility, bad for printing, and bad for SEO. This should be an H1 (but should look the same).
